SimplyPluralApiBodies.yml

```diff
@@ -120,19 +120,34 @@ components:
           description: Description of the channel
           type: string
 
-    chatCategory:
+    chatChannelInfo:
       type: object
       properties:
         name:
-          description: Category name
+          description: Channel name
           type: string
         desc:
-          description: Description of the category
+          description: Description of the channel
           type: string
-        channels:
-          items:
-            type: string
-          description: "List of channel IDs this category contains"
+        uID:
```

Comment:
uID is written as uid, not uID. But why add the uid to this api response specifically? No other api response defines uid